samples/Exporters/AspNet/Global.asax.cs

```diff
@@ -24,6 +27,16 @@ protected void Application_Start()
                     .AddDependencyInstrumentation();
             });
 
+            TracerFactoryBase.SetDefault(this.tracerFactory);
+
+            this.openTelemetry = OpenTelemetrySdk.EnableOpenTelemetry(
+                (builder) => builder.AddDependencyInstrumentation()
+                .UseJaegerActivityExporter(c =>
```

Comment:
@pjanotti FYI I updated to use JaegerActivityExporter since it was merged while I was working on this.